implementation/visualisation/src/main/java/org/inaetics/dronessimulator/visualisation/Settings.java

```diff
@@ -1,8 +1,5 @@
 package org.inaetics.dronessimulator.visualisation;
 
-/**
- * Created by langstra on 10-3-17.
- */
 public class Settings {
```

Comment:
It is already in the visualisation package, shouldn't that be enough? If it is a convention, then sure, it can be refactored.